docs-markdown/src/test/suite/helper/metadata-completion.test.ts

```diff
@@ -63,7 +64,8 @@ export const context: ExtensionContext = {
 		forEach: () => {},
 		clear: () => {},
 		delete: () => {}
-	}
+	},
+	extensionMode: ExtensionMode.Test
```

Comment:
interesting, what does this do?

Reply:
Hey Brian,

Here's some [background info](https://github.com/microsoft/vscode/issues/95926) and the description from the docs.

![image](https://user-images.githubusercontent.com/11825888/88217367-4d353900-cc13-11ea-8870-5637cb059487.png)